src/routes/index.js

```diff
@@ -5,9 +5,12 @@ const router = express.Router();
 const helloWorldController = require('../controllers/hello-world');
 const heroController = require('../controllers/hero-list');
 const registrationController = require('../controllers/registrationController');
+const heroPostController = require('../controllers/heroPostController');
 
 router.get('/helloworld', helloWorldController.helloWorldController);
 
+router.post('/hero', heroPostController.heroPostController);
+
 router.get('/heroes', heroController.getHeroes);
```

Comment:
You created a new controller, there was no reason for it, you should use the existing controller with the same topic.

Reply:
Fixed, thank you